### DISCUSSION TITLE
Версия 2.2.2 не качает с mintmanga

### DISCUSSION THREAD

**Discussion by angry-master-yoda** [lirix360/ReadmangaGrabber#96] [General]
Проверено на:
https://1.mintmanga.one/moskva_2160
https://1.mintmanga.one/blajenstvo___pogibel_bogov

Список глав получаю.
В избранное добавляется.

При попытке скачать граббер почему-то идёт на 24.mintmanga.one, хотя ни одна из ссылок туда не указывает. Ну а там ничего не выходит.

PS По ссылке https://24.mintmanga.one/moskva_2160 даже списка глав не видит.

**Comment by HaruKunOFF** [comment]
Присоединяюсь к проблеме

**Reply by HaruKunOFF** [reply]
Хотя с mintmanga.com вроде качает, но ранее он не работал

**Comment by lirix360** [comment]
В версии 2.2.3 внесены изменение в процесс скачивания с RM/MM, должно работать стабильнее.

**Comment by angry-master-yoda** [comment]
Подтверждаю: в 2.2.3 всё скачивается.
